common/sgx/verifier.c

```diff
@@ -933,17 +933,18 @@ oe_result_t oe_sgx_verify_evidence(
             claims_length));
 
         // Avoid running the loop unless traces are actually generated
-        if (oe_get_current_logging_level() >= OE_LOG_LEVEL_INFO)
+
+        OE_TRACE_INFO("extracted %lu claims", *claims_length);
+        for (size_t i = 0; i < *claims_length; i++)
```

Comment:
why dont we use oe_get_current_logging_level instead of using OE_LOG_LEVEL_INFO

Reply:
We were previously checking for current logging level,because we were using oe_hex-dump which would print to stdout and didn't have a log level based filtering. With oe_hex_dump_trace() we don't have to worry about that.